### PR TITLE
MUMUP-2406 Implement a uw-frame skin for UW Stevens Point.

### PR DESCRIPTION
**Pull request by jiayinjx**

### CODE REVIEW

File: uw-frame-components/css/themes/uw-stevens-point.less [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@import "common-variables.less";
+@import "uw-stevens-point-variables.less";
+@import "../angular.less";
```
Comment (timlevett) [inline uw-frame-components/css/themes/uw-stevens-point.less]: Please move `@import "../angular.less";` to line one of this file.

File: uw-frame-components/css/themes/uw-stevens-point-variables.less [hunk 2]
```diff
@@ -0,0 +1,16 @@
+/* UW Stevens Point colors */
+@color1: #512698; 
+@color2: lighten(@color1, 5%);
+@color3: darken(@color1, 5%);
+@color4: #E1DCCC;
```
Comment (jhanstra) [inline uw-frame-components/css/themes/uw-stevens-point-variables.less]: Have we confirmed these are the right colors?

**Comment by jhanstra** [comment]
Looks great! Nice job @jiayinjx

**Comment by timlevett** [comment]
:+1:

File: uw-frame-components/css/themes/uw-stevens-point-variables.less (hunk 2 above)
Comment (timlevett) [inline uw-frame-components/css/themes/uw-stevens-point-variables.less]: Only color1 was specified in the design subtask. Lets go with this and see what it looks like. I'm not sure how color4 is used.

**Comment by jiayinjx** [comment]
uw-stevens-point_skin
Screenshot as need.

**Pull request merged**